src/openvic-simulation/map/Map.cpp

```diff
@@ -587,7 +605,9 @@ bool Map::_generate_province_adjacencies() {
 
 	for (size_t y = 0; y < height; ++y) {
 		for (size_t x = 0; x < width; ++x) {
-			Province* cur = get_province_by_index(province_shape_image[x + y * width].index);
+			const uint16_t province_index = province_shape_image[x + y * width].index;
+			Province* cur = get_province_by_index(province_index);
+
```

Comment:
Can you revert this please as this PR doesn't affect this function.

Reply:
Done